api/v1beta1/proposal_types.go

```diff
@@ -105,9 +105,11 @@ type ProposalSource struct {
 	// +optional
 	UnarchiveChannel *UnarchiveChannel `json:"unarchiveChannel,omitempty"`
 	// +optional
-	DeployChaincode *DeployChaincodeProposal `json:"deployChaincode,omitempty"`
+	DeployChaincode *DeployChaincode `json:"deployChaincode,omitempty"`
 	// +optional
-	UpgradeChaincode *DeployChaincodeProposal `json:"upgradeChaincode,omitempty"`
+	UpgradeChaincode *DeployChaincode `json:"upgradeChaincode,omitempty"`
+	// +optional
+	UpdateChannelMember *UpdateChannelMember `json:"updateChannelMember,omitempty"`
```

Comment:
will it be betteri if we use `AddChannelMember` instead of `UpdateChannelMember`